### PR TITLE
Remove the massive console print and log from CalciteSqlCompilerTest

### PR DESCRIPTION
**Pull request by Jackie-Jiang**

### CODE REVIEW

File: pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java [hunk 1]
```diff
@@ -339,60 +336,23 @@ public void testStringLiteral() {
     Assert.assertEquals(groupbyList.get(1).getFunctionCall().getOperands().get(1).getIdentifier().getName(), "FOO");
   }
 
-  @Test
-  public void testConverter()
```
Comment (mayankshriv) [inline pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java]: While it makes sense to remove the logging, here's a prior conversation on the intention for this test.

https://github.com/apache/incubator-pinot/pull/4387#discussion_r300516547

File: pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java [hunk 2]
```diff
@@ -339,60 +336,23 @@ public void testStringLiteral() {
...
-      throws IOException {
-    CalciteSqlParser.compileToPinotQuery("SELECT MIN(div(DaysSinceEpoch,2)) FROM mytable");
```
Comment (kishoreg) [inline pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java]: should we add these queries to the sql_queries.list? Its possible that these were added to test edge cases.

Comment (mayankshriv) [reply pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java]: As per the discussion here #4387, this was added as a sample for dev's to be able to debug. I had made the same comment to @Jackie-Jiang above.
